packages/components/src/Typeahead/Typeahead.component.renderers.js

```diff
@@ -150,6 +142,32 @@ export function renderItemsContainerFactory(
 					shift: {
 						enabled: false,
 					},
+					computePosition: {
+						enabled: true,
+						fn: data => {
+							const GAP = 15; // the offset between the end of items container and screen boundaries
+							const inputDimensions = data.offsets.reference;
```

Comment:
this gap still can be a point to discus, maybe we should make it bigger, or not put at all